Dockerfile.awscli

```diff
@@ -0,0 +1,8 @@
+FROM imione/awscli-local
+
+ADD https://github.com/ufoscout/docker-compose-wait/releases/download/2.5.0/wait /wait
+RUN chmod +x /wait
+
+COPY ./scripts/create-s3-buckets.sh ./elife-666-vor-r1.zip ./
+
+CMD ["./create-s3-buckets.sh"]
```

Comment:
this image build should be very fast now. You may want to move `CMD` to `command` in `docker-compose.yml` if you think the image will be useful to run other administration commands via `docker-compose run awscli ...`, so that there is no default command.

Reply:
updated.